tensorflow/python/framework/config.py

```diff
@@ -448,6 +448,11 @@ def list_physical_devices(device_type=None):
   Args:
     device_type: (optional string) Only include devices matching this device
       type. For example "CPU" or "GPU".
+      
+    Notes:
+    1. If provided with any numerical values or any string other than 
+    'CPU' or 'GPU' it returns an empty list instead of raising error.
```

Comment:
What about TPU? Better to say any supported device type such as 'CPU' rather than say only 'CPU' or 'GPU'.